bokeh/models/formatters.py

```diff
@@ -282,26 +283,7 @@ def from_coffeescript(cls, code, args={}):
 
     """)
 
-DEFAULT_DATETIME_FORMATS = lambda : {
-    'microseconds': ['%fus'],
-    'milliseconds': ['%3Nms', '%S.%3Ns'],
-    'seconds':      ['%Ss'],
-    'minsec':       [':%M:%S'],
-    'minutes':      [':%M', '%Mm'],
-    'hourmin':      ['%H:%M'],
-    'hours':        ['%Hh', '%H:%M'],
-    'days':         ['%m/%d', '%a%d'],
-    'months':       ['%m/%Y', '%b%y'],
-    'years':        ['%Y'],
-}
-
-class DatetimeTickFormatter(TickFormatter):
-    """ Display tick values from a continuous range as formatted
-    datetimes.
-
-    """
-
-    formats = Dict(Enum(DatetimeUnits), List(String), default=DEFAULT_DATETIME_FORMATS, help="""
+DATETIME_TICK_FORMATTER_HELP="""
```

Comment:
@bokeh/dev, I didn't update this text, so someone with better English writing capabilities could rewrite this to reflect changes in this PR.